problem-18/README.md

```diff
@@ -0,0 +1,12 @@
+# Task 18: Create Auth Guard to protect admin page (lv 3)
+
+## Description
+Update the 'AdminGuard' to redirect the user when the visit the URL /admin to the /denied route. Then add this guard to the /user route. 
```

Comment:
Guard navigations should depends on a value, whats that value or use case here ?

Reply:
Problem now uses a query param to see if user has access